src/js/events/ChronosActions.js

```diff
@@ -46,10 +55,105 @@ const ChronosActions = {
     },
     {delayAfterCount: Config.delayAfterErrorCount}
   ),
+
+  fetchJobDetail: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      data: [
+        {name: 'embed', value: 'activeRuns'},
+        {name: 'embed', value: 'schedules'}
+      ],
+      success: function (response) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_SUCCESS,
+          data: response,
+          jobID
+        });
+      },
+      error: function (xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          jobID,
+          xhr
+        });
+      },
+      hangingRequestCallback: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DETAIL_ONGOING,
+          jobID
+        });
+      }
+    });
+  },
+
+  deleteJob: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
+      method: 'DELETE',
+      success: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DELETE_SUCCESS,
+          jobID
+        });
+      },
+      error: function (xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_DELETE_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          jobID,
+          xhr
+        });
+      }
+    });
+  },
+
+  runJob: function (jobID) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/jobs/${jobID}/runs`,
+      method: 'POST',
+      data: {},
+      success: function () {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_RUN_SUCCESS
+        });
+      },
+      error: function (xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_CHRONOS_JOB_RUN_ERROR,
+          data: RequestUtil.parseResponseBody(xhr),
+          xhr
+        });
+      }
+    });
+  },
+
+  suspendJob: function (jobID, data) {
+    RequestUtil.json({
+      url: `${Config.rootUrl}/chronos/jobs/${jobID}`,
```

Comment:
Most of this makes sense, except I am not sure what I need to do to properly implement this "suspend" feature.

How do I assemble the schedule's configuration from the `job` data? Since the API doesn't support `PATCH` it sounds like I need to give it the complete configuration with `enabled` set to `false`.